dream-num/univer
# feat(example): add doc uniscript example

- [x] I am sure that the code is update-to-date with the `dev` branch.




close #



- [x] add doc uniscript demo
- [x] add doc appendText api for facade


test: 
1. open doc-uniscript demo
2. open uniscript panel
3. paste this code `univerAPI.getActiveDocument()?.appendText('Hello, Univer!');`
4. excute script
5. check the document content change
